packages/lpa-questionnaire-web-app/src/views/includes/head.njk

```diff
@@ -2,11 +2,10 @@
   {% if appealReplyId %}
       <meta name="appeal-reply-id" content="{{ appealReplyId }}">
     {% endif %}
-  <!--[if lte IE 8]><link href="/public/stylesheets/main-ie8.css" rel="stylesheet" type="text/css" /><![endif]-->
-  <!--[if gt IE 8]><!--><link href="/public/stylesheets/main.css" media="all" rel="stylesheet" type="text/css"/><!--<![endif]-->
+  <link href="/public/stylesheets/main.css" media="all" rel="stylesheet" type="text/css"/>
 
   <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script data-cy="Google Analytics" async src="https://www.googletagmanager.com/gtag/js?id={{ googleAnalyticsId | safe }}"></script>
+  <script data-cy="Google Analytics" async src="https://www.googletagmanager.com/gtag/js?id={{ googleAnalyticsId | safe }}" integrity="sha384-cOKpOdtM/dNut8NuGTs1Tmg/++xJ7lDRHGXSyBmbTzTivwJ4JyWIgKxaUurhvN69" crossorigin="anonymous"></script>
```

Comment:
What's this integrity tag?

Reply:
@swdpcomputing This is the hash of https://www.googletagmanager.com/gtag/js?id={{ googleAnalyticsId | safe }}